README.md

```diff
@@ -1,3 +1,60 @@
 # hubdata
 
 Python tools for accessing and working with hubverse Hub data
+
+## running commands locally
+
+All commands should be run from this repository's root, i.e., first do:
+
+```bash
+cd /<path_to_repos>/hub-data/
+```
+
+### app
+
```

Comment:
Can we maybe add more information here (before `### app`) on how to set up everything properly to be able to run the command below?

Reply:
Also can we please add a little more details/text on what each command does/means? To help the user and give a little more context.

Reply:
Done! Good call.